content/en/opentelemetry/interoperability/drop_in_support.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Using OpenTelemetry Instrumentations with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel drop-in instrumentation']
+---
+
+Instrumentation is the act of adding observability code to an application. Automatic instrumentation is a way to instrument applications and libraries without touching their source code. Both OpenTelemetry and Datadog provide instrumentations as part of their SDKs, with varying amounts of coverage between them.
```

Comment:
```suggestion
Instrumentation is the act of adding observability code to an application. Automatic instrumentation is a way to instrument applications and libraries without directly modifying their source code. Both OpenTelemetry and Datadog provide instrumentations as part of their SDKs, with varying amounts of coverage between them.
```

Reply:
Can provide links for 
Instrumentation: https://docs.datadoghq.com/tracing/trace_collection/

Automatic instrumentation: https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/?tab=singlestepinstrumentationbeta

Reply:
Aside, it'd be good to align definitions between those pages and our glossary

https://docs.datadoghq.com/tracing/glossary/

Reply:
I think the description here does a good job of describing instrumentation libraries, which could be a good basis for our description here: https://opentelemetry.io/docs/concepts/instrumentation/libraries/

Reply:
I added a meta-comment to Ayan's PR and I think it may also apply here https://github.com/DataDog/documentation/pull/23787/files#r1664958998